### ISSUE TITLE
DOMException: Failed to execute 'setItem' on 'Storage': Setting the value of 'gap/gun/' exceeded the quota.

### ISSUE THREAD

**Issue by Jack-Works** [amark/gun#1016]
Maybe gun set too many data in the storage?

**Comment by Tedko** [comment]
@amark  gun is magic!

**Comment by amark** [comment]
@Jack-Works yeah localStorage has 5MB limit, you'll need to try using IndexedDB via https://gun.eco/docs/RAD#install tho I recommend talking to @mmalmi or @sirpy who use IndexedDB in-production with real users, as they might have some edge cases & options they can warn about or recommend.

Note! New GUN (with high performance) that I've been tweeting about is **not** on master or NPM published yet. If you can link me to your `/stats.html` pages we can monitor performance & see if we need to try to upgrade you early (tho sirpy has already run into some issues) before the unit tests are all passing, depending on what is needed urgently or not.

**Comment by Jack-Works** [comment]
Oh I didn't ever learned that gun can use indexed db, I though RAD is for Nodejs.

Does gun able to migrate data in the data in the local storage to indexed db?

And what if it stored too many data in the indexed db?

**Comment by amark** [comment]
@Jack-Works unfortunately not. That is a good/important feature for the future, but pretty low priority for me (trying to keep these 40K users/second in-production peers afloat!)

IndexedDB may ask browser user fro more storage (depending on which browser), after that is used up, you'll get errors.

**Comment by Jack-Works** [comment]
Yeah, does gun able to recover from those errors? For example, use LRU to clean data
